src/effects/onLoaded/test.js

```diff
@@ -0,0 +1,46 @@
+import Immutable from 'seamless-immutable';
+
+import createAction from '../../creators/createAction';
+import createReducer from '../../creators/createReducer';
+
+import onLoaded from '.';
+
+const initialState = {
+  target: null,
+  targetLoading: true,
+  count: 0
+};
+
+const setUp = {
+  state: null
+};
+
+beforeEach(() => {
+  setUp.state = Immutable(initialState);
+});
+
+describe('onLoaded', () => {
+  it('Sets correctly loading target', () => {
+    const reducer = createReducer(setUp.state, {
+      '@@ACTION/TYPE': onLoaded()
+    });
+    const newState = reducer(setUp.state, createAction('@@ACTION/TYPE', 'target'));
```

Comment:
I'd dispatch a onLoading action first, to make sure that targetLoading goes to true first, and then test than after that, sending an onLoaded action sets it to false

Reply:
Or set targetLoading to true in the initial state